### PR TITLE
perf: caching, format updates

### PR DESCRIPTION
**Pull request by TobiTenno**
### What did you fix? 
caching/format updates

1. Drop nexus lookup that hasnt worked due to upstream unavailability
2. Update ci & linting to be consistent with other wfcd node projects

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[No]**
- Have I run the linter? **[Yes]**
- Is is a bug fix, feature request, or enhancement? **[Docs/Maintenance]**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - main
```
Comment (AyAyEm) [inline .github/workflows/release.yaml:6]: Any need to include a main branch which does not exist?

File: index.js [hunk 2]
```diff
@@ -120,15 +88,13 @@ module.exports = class PriceCheckQuerier {
    * @returns {Promise<string>} a Promise of a string containing the results of the query
    */
   async priceCheckQueryString(query, priorResults, platform = 'pc') {
-    const components = priorResults || await this.priceCheckQuery(query, platform);
+    const components = priorResults || (await this.priceCheckQuery(query, platform));
```
Comment (AyAyEm) [inline index.js:91]: Maybe it's not much relevant, but is there any need for the additional parenthesis?

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -1,6 +1,5 @@
-name: Actions
+name: CI
 on:
-  push:
```
Comment (AyAyEm) [inline .github/workflows/ci.yaml:3]: You could also make this run on push to master and later require it on the release workflow that would eliminate the need to duplicate the jobs in both workflows.
[Here's a question on stackoverflow about it](https://stackoverflow.com/questions/58457140/dependencies-between-workflows-on-github-actions)

File: .github/workflows/docs.yaml [hunk 4]
```diff
@@ -10,9 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-      - run: npm i
-      - run: npm i ink-docstrap jsdoc
+        uses: actions/checkout@v3
+      - run: npm ci
+      - run: npm i ink-docstrap jsdoc taffydb
```
Comment (AyAyEm) [inline .github/workflows/docs.yaml:15]: I'm curious, would taffydb be an optional dependency for jsdoc and ink-docstrap?

**Review by AyAyEm — CHANGES_REQUESTED** [review]
Great work, just need to change the title and the pr description there's too much change to be classified as a `ci` change also remember to edit the project description after this pr is merged

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (AyAyEm) [reply .github/workflows/ci.yaml:3]: or better yet make ci a reusable workflow as described [here](https://docs.github.com/en/actions/using-workflows/reusing-workflows)

File: .github/workflows/release.yaml (hunk 1 above)
Comment (TobiTenno) [reply .github/workflows/release.yaml:6]: Yeah, it was always planned to move to main, so ive used the same config where I can

File: index.js (hunk 2 above)
Comment (TobiTenno) [reply index.js:91]: I think.this one is a prettier enforcement

File: .github/workflows/docs.yaml (hunk 4 above)
Comment (TobiTenno) [reply .github/workflows/docs.yaml:15]: Its supposed to be, but the docs generation & deploy fails without it.

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (TobiTenno) [reply .github/workflows/ci.yaml:3]: so, trying to make sure i understand what you're saying, this is basically what it was. i'm almost certain what i had before did about the same, but without splitting them was running extra actions that i'm trying to exclude. i think making a better ci reusable workflow is something i want to do that i will probably safe as a larger overhaul and put it over in https://github.com/WFCD/eslint-config so it's more easily shareable

**Review by AyAyEm — APPROVED** [review]

**Pull request merged**